universal-application-tool-0.0.1/conf/application.conf

```diff
@@ -142,12 +142,12 @@ play.http {
   # ~~~~~
   session {
     # Sets the cookie to be sent only over HTTPS.
-    #secure = true
+    secure = true
 
     # Sets the cookie to be accessed only by the server.
-    #httpOnly = true
+    httpOnly = true
 
-    sameSite = null
```

Comment:
I believe that this will change the behavior of login - if you want to do this, you'll need to check both login providers in staging.

Reply:
Good call, I'll test it after merging. My intention is not to change behavior but to just be more explicit. According to [MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie/SameSite#values) the default value is 'Lax' when unspecified.

Reply:
Yeah, I recall finding that wasn't true when I added this line for the first time.

Reply:
Leaving out this change for this PR